File: app/src/main/java/com/neocaptainnemo/cv/LocaleService.java
package com.neocaptainnemo.cv;

import android.content.Context;
import android.os.Build;

import java.util.Locale;

public class LocaleService {

    private static final LocaleService instance = new LocaleService();

    private LocaleService() {
        //do nothing
    }

    public static LocaleService getInstance() {
        return instance;
    }

    public String getLocale(Context context) {
        Locale currentLocale;
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
            currentLocale = context.getResources().getConfiguration().getLocales().get(0);
        } else {
            currentLocale = context.getResources().getConfiguration().locale;
        }

        if (currentLocale.getLanguage().equals("ru")) {
            return "ru";
        } else {
            return "en";
        }

    }

}

File: app/src/main/java/com/neocaptainnemo/cv/services/LocaleService.kt
package com.neocaptainnemo.cv.services

import android.content.Context
import android.os.Build
import java.util.*
import javax.inject.Inject

class LocaleService @Inject constructor(private val context: Context) : ILocaleService {

    override val locale: String
        get() {

            val currentLocale: Locale = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
                context.resources.configuration.locales.get(0)
            } else {
                context.resources.configuration.locale
            }

            return if (currentLocale.language == "ru") {
                "ru"
            } else {
                "en"
            }
        }


}
